exec/jdbc/src/test/java/org/apache/drill/jdbc/test/Bug1735ResultSetCloseReleasesBuffersTest.java

```diff
@@ -69,23 +69,19 @@ public static void tearDownClass() {
   public void test() throws Exception {
     withNoDefaultSchema()
     .withConnection(
-        new Function<Connection, Void>() {
-          public Void apply( Connection connection ) {
-            try {
-              Statement statement = connection.createStatement();
-              ResultSet resultSet = statement.executeQuery( "USE dfs.tmp" );
-              // TODO:  Purge nextUntilEnd(...) and calls when remaining fragment
-              // race conditions are fixed (not just DRILL-2245 fixes).
-              // resultSet.close( resultSet );
-              statement.close();
-              // connection.close() is in withConnection(...)
-              return null;
-            } catch ( SQLException e ) {
-              throw new RuntimeException( e );
-            }
+        (Function<Connection, Void>) connection -> {
```

Comment:
Possibly rename class name, Test should in the beginning of the name
Test annotation, proper test name